ribs/archives/_archive_base.py

```diff
@@ -155,6 +160,13 @@ def __init__(self,
                                       dtype=self.dtype)
         self._metadata_arr = np.empty(self._cells, dtype=object)
 
+        # For CMA-MAE
+        self._learning_rate = learning_rate
+        # TODO use np.empty
```

Comment:
What's wrong with np.full?

Reply:
I thought I read somewhere that np.empty is preferred over np.full.

From the numpy documentation:
> empty, unlike zeros, does not set the array values to zero, and may therefore be marginally faster. On the other hand, it requires the user to manually set all the values in the array, and should be used with caution.

I think np.full is similar to np.zeros in this sense.